pennylane/beta/queuing/measure.py

```diff
@@ -49,7 +50,7 @@ def __init__(self, return_type, obs=None, wires=None, eigvals=None):
         self.return_type = return_type
         self.obs = obs
 
-        self._wires = None
+        self._wires = wires or Wires([])
```

Comment:
Does this affect line 66?

Reply:
Good question, I've now tidied: dc7b927af875cb3b47dc1456d4be3663b62847c2